src/coffee/services/product-projections.coffee

```diff
@@ -256,6 +344,12 @@ class ProductProjectionService extends BaseService
     _.each searchKeywords, (keys) ->
       customQueryString.push "searchKeywords.#{keys.lang}=#{keys.text}"
 
+    # priceSelection params
+    customQueryString.push "priceCurrency=#{priceCurrency}" if priceCurrency
```

Comment:
`priceCurrency` is required to enable price selection, can we throw if `priceCurrency` is not set